.github/workflows/build.yml

```diff
@@ -95,7 +98,39 @@ jobs:
       - name: Upload Package
         uses: actions/upload-artifact@v3
         with:
-          name: macos-package_${{ matrix.arch }}
+          name: massastation_installer_darwin_${{ matrix.arch }}
           path: "installer/*.pkg"
           if-no-files-found: error
           retention-days: 7
+
+  build-packages-windows:
+    name: Build .msi package for amd64
+    needs: build
+    defaults:
+      run:
+        working-directory: ./installer
+
+    runs-on: windows-latest
```

Comment:
`runs-on: windows-latest` it's not a good practice to rely on latest versions (same for macos). Is there a specific reason?

Reply:
There is no real reason for that.
Do you think I should use a specific version ? Or is the latest windows version fine (I'm talking about windows-2022) ?

Reply:
@peterjah WDYT?

Reply:
No, we should use lastest.